lib/student.rb

```diff
@@ -1 +1,22 @@
+class Student
+  attr_reader :name,
+              :age,
+              :scores
 
+  def initialize(student_attributes)
+    @name = student_attributes[:name]
+    @age = student_attributes[:age]
+    @scores = []
+  end
+
+  def log_score(score)
+    @scores << score
+  end
+
+  def grade
+    sum = scores.sum
+    count = scores.count
+    grade = sum.to_f / count.to_f
+    grade
+  end
```

Comment:
I think this would work exactly the same way:
```suggestion
  def grade
    sum = scores.sum
    count = scores.count
    sum.to_f / count.to_f
  end
```